cmd/flow-dps-indexer/main.go

```diff
@@ -49,18 +49,32 @@ func run() int {
 
 	// Command line parameter initialization.
 	var (
-		flagCheckpoint string
-		flagData       string
-		flagForce      bool
-		flagIndex      string
-		flagLevel      string
-		flagTrie       string
+		flagCheckpoint        string
+		flagData              string
+		flagForce             bool
+		flagIndex             string
+		flagIndexAll          bool
+		flagIndexBlocks       bool
+		flagIndexEvents       bool
+		flagIndexHeaders      bool
+		flagIndexPayloads     bool
+		flagIndexRegisters    bool
+		flagIndexTransactions bool
+		flagLevel             string
+		flagTrie              string
 	)
 
 	pflag.StringVarP(&flagCheckpoint, "checkpoint", "c", "", "checkpoint file for state trie")
 	pflag.StringVarP(&flagData, "data", "d", "", "database directory for protocol data")
 	pflag.BoolVarP(&flagForce, "force", "f", false, "overwrite existing index database")
 	pflag.StringVarP(&flagIndex, "index", "i", "index", "database directory for state index")
+	pflag.BoolVarP(&flagIndexAll, "index-all", "a", false, "index everything")
```

Comment:
In #196 @awfm9 suggested the opposite. I agree that it seems more intuitive for me that by default we'd index everything, but he has more perspective than I do on the use case so it probably makes more sense this way

Reply:
Ah, okay, I read the discussion now. Yeah, even though not the most intuitive, it is the safest way, since it explicitly requires the user to say what he wants.